PyIlmBase/CMakeLists.txt

```diff
@@ -106,7 +105,6 @@ endif()
 # let's run a search and see what we get instead of making it
 # an explicit required. The older names are not portable, but
 # we'll do the best we can
```

Comment:
Boost_NO_BOOST_CMAKE protects the build from the FindBoost.cmake from failing unnecessarily, as I think was the solution for one of the MinGW issue threads. Will the advice be to set it from the command line if one is in a sufficiently off-the-beaten-path boost environment? Is there any reason not to retain Boost_NO_BOOST_CMAKE here?

Reply:
Yes, I wanted to test with and without it. I am going to leave it on (about to restore it), as I think it fixes more than it breaks. I'm also adding a note in the INSTALL.md in the prerequisites section and in the error message I print out about this in the CMakeLists.txt hoping it helps people find that we are trying and maybe give them a clue to look for